joomla/joomla-cms
# [4.1] Allow Save As Copy Article In The Frontend

Pull Request for Issue # .

### Summary of Changes
This is a redo of RTC PR https://github.com/joomla/joomla-cms/pull/32520 . When you edit an article from frontend of your site, there will be a new button **Save As Copy** allows you to create a new article from data of that being edited article.

### Testing Instructions
1. Apply Patch.
2. Login to frontend of your site using an account which has permission to edit an existing article (could be super user account)
3. Edit an existing article. Look at the Bottom of the page, if you are using an account which is having **Create** Article Permission, you will see a new button **Save As Copy**.
4. Press **Save As Copy** button, confirm that the article is successfully created.
